subfun/design/getMockConfig.m

```diff
@@ -1,6 +1,9 @@
 function cfg = getMockConfig(isMT)
     % fixture
     %
+    % REMINDER: If `cfg.design.nbRepetitions` and `cfg.design.nbEventsPerBlock` values are different
+    %           from 12, tests on GitHub will fail so do not change.
```

Comment:
```suggestion
    % REMINDER: If `cfg.design.nbRepetitions` and `cfg.design.nbEventsPerBlock` values are changed
    %           this may make the tests on GitHub fail, so be careful.
```

avoid "hard coding" values in the comment too, in case we change them in the future. :wink:

Reply:
you right!